forms/TagList/Item/style.scss

```diff
@@ -0,0 +1,23 @@
+.hui-TagListItem {
+  display: inline-block;
+  margin: 8px 8px 8px 0;
+}
+
+.hui-TagListItem__label {
+  background-color: $hui-rule-grey;
+  padding: $hui-x-1 $hui-x-2;
+  border-radius: 6px;
+  color: $hui-primary-grey;
+  font-family: lato,helvetica,arial,sans-serif;
+}
+
+.hui-TagListItem__icon--right {
+  padding-left: $hui-x-2;
+}
```

Comment:
Maybe write the styles for `.hui-TagListItem__icon--left` and then that prop would make sense